pkg/controllers/machine/terminator/eviction.go

```diff
@@ -38,6 +39,26 @@ const (
 	evictionQueueMaxDelay  = 10 * time.Second
 )
 
+type NodeDrainError struct {
+	Err error
```

Comment:
I meant

```
type NodeDrainError struct {
   error
}
```
and then deleting the `Error()` method. 

Does that not work here?